src/components/errorBoundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,48 @@
+import { Component, ErrorInfo, ReactNode } from "react";
+import styles from "./errorBoundary.module.scss";
+import errorImage_dark from "../../assets/img/fatalError_dark.png";
+import errorImage_light from "../../assets/img/fatalError_light.png";
+import i18next from "i18next";
+import classNames from "classnames";
+
+interface Props {
+  children?: ReactNode,
+  theme:string
+}
+
+interface State {
+  hasError: boolean;
+}
+
+class ErrorBoundary extends Component<Props, State> {
+  public state: State = {
```

Comment:
remove public modifier. 
You don't need this methods outside of ErrorBoundary class

Reply:
fixed